README.md

```diff
@@ -201,6 +207,95 @@ const rule = {
 };
 ```
 
+### Includes
+
+The includes rule has three distinct behaviors, depending on whether the input is a string, an array, or an object.
+
+##### Input is a String
+
+When input is a string, the includes rule will check if the rule property in the config is a substring of the input.
+
+```yaml
+isSomeFeatureEnabled:
+  values:
+    - value: true
+      name: john
+    - value: false
+```
+
+```javascript
+const rule = {
+  type: 'includes',
+  input: input => input,
+  property: 'name',
+};
+...
+flipr.getValue(isSomeFeatureEnabled, 'john'); // true
+flipr.getValue(isSomeFeatureEnabled, 'johnathan'); // true
```

Comment:
Was there a reason to make string match on substring, but an array of strings matches as exact match? I would expect same behavior in both cases.

Reply:
it's lodash's behavior, we're just using their `_.includes`

Reply:
Could make the string value into an array when passing into `_.includes` to force it to be an exact match. It seems like substring matches could be more likely to have unexpected effects.

Reply:
i think the name of the rule, "includes", is sufficient to explain the existing behavior in this case.  this is how i would expect it to work without looking at the code